.github/workflows/docker-nightly-publish.yml

```diff
@@ -30,102 +40,45 @@ env:
 jobs:
   create-runners:
     runs-on: [ self-hosted, scheduler ]
+    strategy:
+      matrix:
+        arch: ${{ fromJson(inputs.arch) }}
```

Comment:
if a user passes in a single container, does it need to be specified like `["lmi"]`, or will `"lmi"` work as well?